src/components/Header/index.tsx

```diff
@@ -28,7 +28,14 @@ const Header = () => {
     return isVisible ? (
         <div className="flex w-full flex-row items-center justify-between border-b border-gray-300 p-4 pb-2">
             <Link to="/">
-                <div className="text-2xl font-medium">OkeyDoke</div>
+                <div className="flex flex-col items-left justify-between">
```

Comment:
```suggestion
                <div className="flex flex-col items-start justify-between">
```

`items-left` is not a tailwind className lmao.